src/components/InternationalComparison.tsx

```diff
@@ -84,6 +86,16 @@ export const InternationalComparison = ({ country, matchPercentage, variant }: P
       <div style={{ display: 'flex' }}>
         <h3 style={{ flexGrow: 1 }}>{variant.name ?? 'Unnamed Variant'} - International Comparison</h3>
         <div>
+          {AccountService.isLoggedIn() && (
+            <Button
+              onClick={() => NextcladeService.showVariantOnNextclade(variant, matchPercentage, undefined)}
+              variant='outline-dark'
+              size='sm'
+              style={{ marginRight: '10px' }}
```

Comment:
You can also use `className="mr-1"` (or `mr-2`, [see here](https://getbootstrap.com/docs/5.0/utilities/spacing/)) I know it's not much nicer, but if we switch to a Bootstrap theme with different spacing, then the spacing that these utility classes make will adjust automatically.